jenkins/job-configs/kubernetes-jenkins-pull/kubernetes-pull.yaml

```diff
@@ -51,6 +51,7 @@
         - github-pull-request:
             # This is the Jenkins GHPRB plugin ID, not the actual github token.
             auth-id: 'f8e31bc1-9abb-460a-a2ca-9c4aae3ca4e8'
+            only-trigger-phrase: {only-trigger-phrase}
```

Comment:
single quotes should not be required (I see other fields here without quotes, `status-context` for ex).
But this did not work, it resulted in `<onlyTriggerPhrase>ordereddict([('only-trigger-phrase', none)])</onlyTriggerPhrase>`. Will try adding quotes.